server/src/main/scala/services/graphstages/WorkloadCalculator.scala

```diff
@@ -63,8 +63,11 @@ object WorkloadCalculator {
         case UndefinedSplitStyle => splitsToUse.splits.map(qc => qc.copy(paxCount = 0))
         case PaxNumbers =>
           val splitsWithoutTransit = splitsToUse.splits.filter(_.queueType != Queues.Transfer)
-          val totalSplitsPax = splitsWithoutTransit.toList.map(_.paxCount).sum
-          splitsWithoutTransit.map(qc => qc.copy(paxCount = qc.paxCount / totalSplitsPax))
+          val totalSplitsPax: Load = splitsWithoutTransit.toList.map(_.paxCount).sum
```

Comment:
we should probably add a test for this.

Reply:
yes and guard against empty splits as you can't sum a Nil